# Speed Units.
# Meters per second.
#uint8 SUNITS_METERS_PS = 0
# RPM.
#uint8 SUNITS_RPM = 1
# Percentage.
#uint8 SUNITS_PERCENTAGE = 2

std_msgs/Float64 value
std_msgs/UInt8 speed_units
